R/store.R

```diff
@@ -0,0 +1,43 @@
+global_store <- new.env(parent = emptyenv())
+
+store_start <- function(global = TRUE) {
+  if (!global || is.null(global_store$store)) {
+    message("connecting to redis at ", redux::redis_config()$url)
+    con <- redux::hiredis()
+
+    message("Starting store")
+    store <- redux::storr_redis_api(prefix = "store", con = con)
+
+    global_store$store <- store
+  }
+  invisible(global_store$store)
+}
+
+store_set <- function(key, data) {
+  global_store$store$set(key, data)
+}
+
+refresh_store <- function() {
+  if(!is.null(global_store$store) && length(global_store$store$list()) > 0) {
+    global_store$store$clear()
```

Comment:
Looks like tests are still unstable. I'm trying to create a way here to make sure I have a fresh store for each test to use. But looks like this keeps running into problems with restarting the store or properly clearing it. Is there a good way to do this? I've tried using `store$destroy()` too but no luck there either.